ui-tests/cypress/support/pageObjects/app/Dialog.js

```diff
@@ -67,6 +67,11 @@ export default class Dialog extends WebElement {
                .children(htmlElements.button).eq(1);
   }
 
+  getFooterActionByLabel(label) {
+    return this.getFooter()
+      .children(htmlElements.button).contains(label);
```

Comment:
Please, do not use `contains` in page objects.
This class already has methods to close, cancel and confirm the operation by the way

Reply:
done